stackslib/src/config/mod.rs

```diff
@@ -2177,6 +2191,8 @@ pub struct MinerConfig {
     pub tenure_extend_cost_threshold: u64,
     /// Define the timeout to apply while waiting for signers responses, based on the amount of rejections
     pub block_rejection_timeout_steps: HashMap<u32, Duration>,
+    /// Define max execution for contract calls
```

Comment:
Can you explain it a bit more in this comment? Assume whatever is here is exactly what will show up in the documentation.